scripts/V53__INSERINDO_PARAMETROS_CARGA_INICIAL.sql

```diff
@@ -0,0 +1,3 @@
+﻿insert
+into carga_inicial (ano, tipos_ue, ues, turmas, criado_em)
+values (2022, '1,2,3,4,10,11,12,13,16,17,18,23,28,31', null, null, current_date);
```

Comment:
O parametro é para 2021
```suggestion
values (2021, '1,2,3,4,10,11,12,13,16,17,18,23,28,31', null, null, current_date);
```